Library/Formula/libswiften.rb

```diff
@@ -1,24 +1,13 @@
 class Libswiften < Formula
   desc "C++ library for implementing XMPP applications"
   homepage "https://swift.im/swiften"
-  revision 2
 
   stable do
-    url "https://swift.im/downloads/releases/swift-2.0/swift-2.0.tar.gz"
-    sha256 "cbcdbe527dc4d112a38f3cdec5f1051d3beba0b97f8082f90debd04a5b45a41f"
+    url "git://swift.im/swift", :tag => "swift-3.0", :revision => "14ddf8b470f5a3420b5f2c96daea33c2513cac6e"
+    depends_on "lua" => :optional
 
-    # Patch to include lock from boost. Taken from
-    # http://comments.gmane.org/gmane.linux.redhat.fedora.extras.cvs/957411
+    # Patch to fix build error of dynamic library with Apple's Secure Transport API
```

Comment:
Thanks. Got a patch ready to go into review as soon as I've checked it builds (looks like Windows was similarly afflicted).